rclpy/rclpy/executors.py

```diff
@@ -571,15 +592,15 @@ def _wait_for_ready_callbacks(
                             yield handler, gc, node
 
                 for client in node.clients:
-                    if client.client_pointer in clients_ready:
+                    if client.handle.pointer in clients_ready:
```

Comment:
Do we not need to get the capsules for `client` here and `server` below?

Reply:
The capsules are only needed when they're used. The timer had to get its capsule to call `rclpy_is_timer_ready` above. The client and service capsules are gotten in `executor._take_client` and `executor._take_service`.